create.js

```diff
@@ -2,11 +2,12 @@
 
 var join = require('path').join
 var EventEmitter = require('events')
-
 var pull = require('pull-stream')
 var ref = require('ssb-ref')
 var ssbKeys = require('ssb-keys')
 
+var createDB = require('./db')
+var extras = require('./extras')
```

Comment:
I pulled these to the top of the file because I like to see what's been pulled into a file
I was passing through here trying to understand where the addUnboxer connected in an how.
I hope this help later people reading to read more easily

Reply:
Very much appreciated, thank you.